--- satis-1.0.0/bin/satis~	2017-08-10 09:25:40.000000000 +0300
+++ satis-1.0.0/bin/satis	2018-03-09 02:05:34.903992878 +0200
@@ -17,7 +17,7 @@
     }
 }
 
-if ((!$loader = includeIfExists(__DIR__.'/../vendor/autoload.php')) && (!$loader = includeIfExists(__DIR__.'/../../../autoload.php'))) {
+if ((!$loader = includeIfExists(__DIR__ . '/../src/Composer/Satis/autoload.php')) && (!$loader = includeIfExists('/usr/share/php/Composer/Satis/autoload.php'))) {
     print('You must set up the project dependencies using Composer before you can use Satis.');
     exit(1);
 }
--- satis-1.0.0/src/Console/Command/BuildCommand.php~	2017-08-10 09:25:40.000000000 +0300
+++ satis-1.0.0/src/Console/Command/BuildCommand.php	2018-03-09 02:07:34.756989967 +0200
@@ -282,7 +282,7 @@
 
             $data = json_decode($content);
 
-            $schemaFile = __DIR__ . '/../../../res/satis-schema.json';
+            $schemaFile = __DIR__ . '/../../res/satis-schema.json';
             $schema = json_decode(file_get_contents($schemaFile));
             $validator = new Validator();
             $validator->check($data, $schema);
--- satis-1.0.0/src/Builder/WebBuilder.php~	2017-08-10 09:25:40.000000000 +0300
+++ satis-1.0.0/src/Builder/WebBuilder.php	2018-03-09 02:08:33.493425654 +0200
@@ -104,7 +104,7 @@
         if (null === $this->twig) {
             $twigTemplate = isset($this->config['twig-template']) ? $this->config['twig-template'] : null;
 
-            $templateDir = $twigTemplate ? pathinfo($twigTemplate, PATHINFO_DIRNAME) : __DIR__ . '/../../views';
+            $templateDir = $twigTemplate ? pathinfo($twigTemplate, PATHINFO_DIRNAME) : __DIR__ . '/../views';
             $loader = new \Twig_Loader_Filesystem($templateDir);
             $this->twig = new \Twig_Environment($loader);
         }
